UI/login.html

```diff
@@ -36,7 +36,15 @@
                     </a>
                 </li>
                 <li>
+<<<<<<< HEAD
```

Comment:
Great job, however you have not properly handled spill over from your git conflict. Kindly clean up this file as it currently affects your UI